server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -121,12 +121,12 @@ export async function createGitService(createArgs: createGitServiceArgs): Promis
 	const maxCacheableSummarySize: number =
 		config.get("restGitService:maxCacheableSummarySize") ?? 1_000_000_000; // default: 1gb
 
-	let isEphemeral = isEphemeralContainer;
+	let isEphemeral = cache ? isEphemeralContainer : false;
```

Comment:
Maybe the `cache` question should be part of the `ignoreEphemeralFlag` logic, not the `isEphemeralContainer` logic?

Reply:
Yes, maybe we could invert the order of checks and bring `ignoreEphemeralFlag` to the outer layer? So if ephemeral should be ignored, then we do not do any processing even if `isEphemeral` is true for that document.

Reply:
Unfortuantely, that will not work because the isEphemeral is being passed to RestGitService and if that is not set to false, we will end up creating a ephemeral FS.